src/server.js

```diff
@@ -79,28 +86,32 @@ class Server extends EventEmitter {
     const client = this.clients[address]
     if (!client) {
       // Ignore packets from clients that are not connected.
-      debug(`ignoring packet from unknown inet addr: ${address}`)
+      debug(`Ignoring packet from unknown inet address: ${address}`)
       return
     }
+
     process.nextTick(() => client.handle(buffer))
   }
 
   getAdvertisement () {
     if (this.options.advertisementFn) {
       return this.options.advertisementFn()
     }
+
     this.advertisement.playersOnline = this.clientCount
     return this.advertisement
   }
 
   async listen (host = this.options.host, port = this.options.port) {
     this.raknet = new this.RakServer({ host, port }, this)
+
     try {
       await this.raknet.listen()
     } catch (e) {
-      console.warn(`Failed to bind server on [${this.options.host}]/${this.options.port}, is the port free?`)
+      console.warn(`Failed to bind server on [${this.options.host}]/${this.options.port}], is the port free?`)
```

Comment:
The bracket should be around the address. It's intended to encapsulate an IPv6 address.

Reply:
oops, i didn't see the other bracket after the hostname